projects/common/lib/components/checkbox/checkbox.component.html

```diff
@@ -1,11 +1,21 @@
 
-<input #checkbox type="checkbox" class="input-lg form-check-input" name="checkbox_{{objectId}}" 
-    id="checkbox_{{objectId}}" [ngModel]='data' 
-    (ngModelChange)='dataChange.emit($event);' 
-    [disabled] = 'disabled' [required]="required">
+<input #checkbox
+      type="checkbox"
+      class="input-lg form-check-input"
+      name="checkbox_{{objectId}}"
+      id="checkbox_{{objectId}}"
+      [ngModel]="data"
+      (ngModelChange)="setCheckboxVal($event);"
```

Comment:
Was this the source of the bug you mentioned?  I see you switched from directly calling `dataChange.emit($event)` to `setCheckboxVal($event)`, which calls the onTouch/onChange NgControl methods but it looks like it works.